spanner/pdml.go

```diff
@@ -88,6 +93,23 @@ func (c *Client) partitionedUpdate(ctx context.Context, statement Statement, opt
 			if err == nil {
 				return count, nil
 			}
+			if isUnimplementedErrorForMultiplexedPartitionedDML(err) && sh.session.pool.isMultiplexedSessionForPartitionedOpsEnabled() {
+				sh.session.pool.disableMultiplexedSessionForPartitionedOps()
```

Comment:
Are we trying to pick a regular session during failure for the transaction to proceed?
I guess this will complicate client lib logic. We should instead fail this transaction and all subsequent transactions using multiplexed sessions can fallback to regular.
Atleast this was how it was implemented for read-write to avoid addition client complexity.

Reply:
@pratickchokhani please help clarify this.

Reply:
For PDML, this is the case as the complication is trival.

Reply:
Added warning log for this fallback as discussed offline.